packaging/systemd/microshift-containerized.service

```
@@ -1,9 +1,9 @@
-# microshift
+# container-microshift.service
 # autogenerated by Podman 3.4.0
-# Thu Oct 28 14:18:51 UTC 2021
+# Fri Nov  5 14:30:39 UTC 2021
 
 [Unit]
-Description=MicroShift Containerized
+Description=Podman container-microshift.service
```

```suggestion
Description=MicroShift Containerized
```

This is what the final user will see, can we keep it ? I know podman will auto-generate the other